host: MacBook-Pro-von-Frederic.local
Building DAG of jobs...
Using shell: /opt/local/bin/bash
Provided cores: 8
Rules claiming more threads will be scaled down.
Job stats:
job             count
------------  -------
combine_exps        1
dataset_cx          1
test_rule           1
total               3

Select jobs to execute...
Execute 1 jobs...

[Sun Dec  8 01:13:10 2024]
localrule dataset_cx:
    input: results/__cythonized__
    output: results/real/epinions/null_model=False/cx/e=False p=False l=10 s=1/run/cx_0.csv
    jobid: 2
    reason: Missing output files: results/real/epinions/null_model=False/cx/e=False p=False l=10 s=1/run/cx_0.csv
    wildcards: dataset=epinions, null_model=False, exact=False, parallel=False, max_length=10, n_samples=1, run=0
    resources: tmpdir=/var/folders/dj/14c985vd1ng7gf0vvw_ytrpw0000gn/T

RuleException:
CalledProcessError in file /Users/fredericbusch/Desktop/Thesis/forks/py-raccoon/workflow/Snakefile, line 127:
Command 'set -euo pipefail;  /Users/fredericbusch/miniforge3/envs/py-raccoon/bin/python3.12 /Users/fredericbusch/Desktop/Thesis/forks/py-raccoon/.snakemake/scripts/tmpcd3so030.experiment.py' returned non-zero exit status 1.
[Sun Dec  8 01:13:12 2024]
Error in rule dataset_cx:
    jobid: 2
    input: results/__cythonized__
    output: results/real/epinions/null_model=False/cx/e=False p=False l=10 s=1/run/cx_0.csv

Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
Complete log: .snakemake/log/2024-12-08T011310.352766.snakemake.log
WorkflowError:
At least one job did not complete successfully.
